osu.Game/Screens/Play/Player.cs

```diff
@@ -297,10 +298,7 @@ private void load(OsuConfigManager config, OsuGameBase game, CancellationToken c
                     {
                         if (!this.IsCurrentScreen()) return;
 
-                        if (PerformExit(false))
-                            // The hotkey overlay dims the screen.
-                            // If the operation succeeds, we want to make sure we stay dimmed to keep continuity.
-                            fadeOut(true);
+                        PerformExit(false, true);
```

Comment:
At the least if `PerformExit()` *has* to has two incoming bool params I'd ask them to be explicitly named in the invocation, because these types of `(true, false, true, false, true)` calls make me want to cry when reading the source outside of rider.